### PR TITLE
Include MySQL DeploymentConfig app in integration test suite

### PR DESCRIPTION
**Pull request by viveksinghggits**
## Change Overview
This PR adds MySQL that is deployed on OpenShift cluster using DeploymentConfig resource into our integration test suite.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #XXX

## Test Plan

Since we cant deploy OpenShift on Containers we can test this app locally only as of now. Make the changes in the Makefile to not run in docker image, and include `MysqlDBDepConfig` in `SHORT_APPS` in `integration-test.sh`, after that just run 

```
make integration-test
```

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: pkg/app/mysql-deploymentconfig.go [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+	if err != nil {
+		return errors.Wrapf(err, "Error installing app on openshift cluster.", field.M{"app": mdep.name, "out": out})
+	}
+
+	err = mdep.createMySQLToolsPod(ctx)
+	if err != nil {
+		return errors.Wrapf(err, "Error creating mysql tools pod")
+	}
+
+	err = mdep.createMySQLSecret(ctx)
```
Comment (PrasadG193) [inline pkg/app/mysql-deploymentconfig.go]: `return mdep.createMySQLSecret(ctx)`

File: pkg/app/mysql-deploymentconfig.go [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+							Name:  "MYSQL_ROOT_PASSWORD",
+							Value: mdep.envVar["MYSQL_ROOT_PASSWORD"],
+						},
+					},
+				},
+			},
+		},
+	}
+
+	_, err := mdep.cli.CoreV1().Pods(mdep.namespace).Create(mysqlToolPod)
```
Comment (PrasadG193) [inline pkg/app/mysql-deploymentconfig.go]: `return err`

File: pkg/app/mysql-deploymentconfig.go [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+			Name:      mdep.name,
+			Namespace: mdep.namespace,
+		},
+		Data: map[string][]byte{
+			"mysql-root-password": []byte(mdep.envVar["MYSQL_ROOT_PASSWORD"]),
+		},
+	}
+
+	_, err := mdep.cli.CoreV1().Secrets(mdep.namespace).Create(mysqlSecret)
+	if err != nil {
```
Comment (PrasadG193) [inline pkg/app/mysql-deploymentconfig.go]: No need to check if err is nil. Same at other places

Comment (viveksinghggits) [reply pkg/app/mysql-deploymentconfig.go]: I wanted to wrap the error detail with the actual message, do you think removing that is better option?

**Pull request merged**